src/logic/operate.js

```diff
@@ -0,0 +1,28 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const numberOneLocal = Big(numberOne);
+  const numberTwoLocal = Big(numberTwo);
+  let returnVal = '';
+
+  switch (operation) {
+    case '+':
+      returnVal = numberOneLocal.plus(numberTwoLocal);
+      break;
+    case '-':
+      returnVal = numberOneLocal.minus(numberTwoLocal);
+      break;
+    case 'X':
+      returnVal = numberOneLocal.times(numberTwoLocal);
+      break;
+    case '÷':
+      returnVal = numberOneLocal.div(numberTwoLocal);
+      break;
```

Comment:
dont forget to manage division by ZERO in the next milestone, else it will generate an error.